compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -1049,7 +1049,8 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             (args, None)
         };
 
-        let mut copied_constant_arguments = vec![];
+        // Keeps track of temporary allocas whose liftime need to be ended after the call.
+        let mut lifetime_ends_after_call: Vec<(Bx::Value, Size)> = Vec::new();
```

Comment:
also comments should generally say "why": probably this?
```suggestion
        // because we want to know to emit things like `@llvm.lifetime.end` for them
        let mut lifetime_ends_after_call: Vec<(Bx::Value, Size)> = Vec::new();
```